tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/parser/ResilientParserTest.java

```diff
@@ -214,4 +214,16 @@ public void testResilientParsingError() {
         BAssertUtil.validateError(result, 1, "missing equal token", 2, 12);
         BAssertUtil.validateError(result, 2, "missing identifier", 2, 12);
     }
+
+    @Test
+    public void testListBindingPatternWithInvalidMember() {
```

Comment:
As the name implies `ResilientParserTest` class is to add tests for the parser resiliency.
Since this test is not related to resiliency and the fix is within the parser, shall we add a parser test case to `TypedBindingPatternsTest` class?

Reply:
Updated